File: C++/two_sum.cpp
class Solution {
public:
    vector<int> twoSum(vector<int>& nums, int target) {
        vector<int> indexes = {}; 
        for(int i = 0; i < nums.size(); i++){
            int first = nums[i];
            for(int j = 0; j < nums.size(); j++){
                int second = nums[j];
                if((j > i)&&(first + second == target)){
                    indexes = {i, j};
                }
            }
            
        }
    return indexes;       
    }
};

File: python/two_sum.py
class Solution(object):
    def twoSum(self, nums, target):
        """
        :type nums: List[int]
        :type target: int
        :rtype: List[int]
        """
        # brute force
        for i in range(len(nums)):
            first = nums[i]
            for j in range(len(nums)):
                second = nums[j]
                if j > i and (first + second) == target:
                    return i, j
                     